presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -265,17 +265,10 @@ public static String textLogicalPlan(PlanNode plan, Map<Symbol, Type> types, Met
         return new PlanPrinter(plan, types, metadata, statsCalculator, costCalculator, session, stats, indent, verbose).toString();
     }
 
-    public static String textDistributedPlan(StageInfo outputStageInfo, Metadata metadata, StatsCalculator statsCalculator, CostCalculator costCalculator, Session session)
-    {
-        return textDistributedPlan(outputStageInfo, metadata, statsCalculator, costCalculator, session, false);
-    }
-
     public static String textDistributedPlan(StageInfo outputStageInfo, Metadata metadata, StatsCalculator statsCalculator, CostCalculator costCalculator, Session session, boolean verbose)
     {
         StringBuilder builder = new StringBuilder();
-        List<StageInfo> allStages = outputStageInfo.getSubStages().stream()
-                .flatMap(stage -> getAllStages(Optional.of(stage)).stream())
-                .collect(toImmutableList());
+        List<StageInfo> allStages = getAllStages(Optional.of(outputStageInfo));
```

Comment:
When running EXPLAIN ANALYZE I believe this is the piece of code that removes the explain node itself from the plan.

Reply:
Yep.  That is why I changed the explain operator to strip the top stage.

Reply:
> Yep. That is why I changed the explain operator to strip the top stage.

be sure to mention that in release notes. We're kind of used to ±1 stage idiosyncrasies in explain/explain analyze/web ui, so any change here is must-to-know-about.

Reply:
This change should not be user visible.  There was only caller of this code, and I changed it to strip the "Explain Analyze" stage of the query (instead of this code doing it implicitly).